Documentation/final-report/pages/technical_solution/machine_learning.tex

```diff
@@ -41,36 +57,35 @@ \subsubsection{YOLO model used for car detection}
 for example RCNN, Fast-RCNN and Faster-RCNN(\cite{singlevstwostagedetectors}).
 
 For the scope of this project, we chose to continue with the YOLO models because
-of their speed, their scalability, their lack of computational resources and
-they are free!
+of their speed, their scalability and their lack of computational resources.
+
+We experimented with different versions of the YOLOv5 model, then slowly
+levelled up the versions up to version 8.  The YOLOv8 model has been pre-trained
+on the COCO (Common Object in Context) dataset, a popular large scale dataset
+with 200,000 images across 80 object categories commonly used for benchmarking
+computer vision models (\cite{cocodataset}).
 
-We moved away from Label studio and studied the Ultralytics documentation and
-tried out the YOLOv5 model to recognise the cars on the satellite images. We
-experimented with different versions of the YOLOv5 model, then slowly levelled
-up the versions up to version 8, monitoring metrics such as mAP (Mean Average
-Precision), F1-score and the false positive rate. The figure below shows the
-metrics of the model as we iteratively experimented up the version chain.
+We monitored metrics such as mAP (Mean Average Precision), F1-score and the
```

Comment:
It should probably be either "MAP" or you should capitalise Average and Precision, but not Mean